src/modules/account/accountApi.js

```diff
@@ -87,7 +88,7 @@ function makeAccountApi (
 
     // Change or create credentials:
     changePassword (password: string): Promise<void> {
-      return state.changePassword(password)
+      return state.changePassword(password).then(() => {})
```

Comment:
What does this anonymous function do?

Reply:
It returns `undefined`, which is what Flow needs us to return.